src/resources/jupyter/lang/julia/setup.jl

```diff
@@ -10,30 +10,49 @@ import IJulia
 # clear console history
 IJulia.clear_history()
 
+fig_width = {0}
+fig_height = {1}
+fig_format = :{2}
+fig_dpi = {3}
+
+# no retina format type, use svg for high quality type/marks
+if fig_format == :retina
+  fig_format = :svg
+elseif fig_format == :pdf
+  fig_dpi = 96
```

Comment:
Do we want the DPI to be left alone here? (we default it to 300 for PDF, we defaulted it back to 96 only b/c we were re-routing to PNG)

Reply:
I see now that we need the dpi to be 96 for the pixel calculation to be correct